Statistics/Distributions/ContinuousDistribution.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Remoting;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Utilities;
+
+namespace Statistics
+{
+    public abstract class ContinuousDistribution : Base.Implementations.Validation, IDistribution
```

Comment:
I'd like to talk about using abstract classes vs interfaces, not sure I totally understand the need to have an abstract class between the interface and the concrete class

Reply:
the issue is that validation is a class i wish to inherit. An alternative would be to inherit Base.Implemetnations.Validation on all implemetnations of Idistribution, but then i wouldnt be certain to have the methods I want. 
This also allows me to implement what was once implemented in the extension class and is a more conventional way of doing that.